### ISSUE TITLE
Possible option to use "VACUUM INTO"

### ISSUE THREAD

**Issue by cpjolicoeur** [efrecon/sqlite-backup#7]
Would you be open to a Pull Request adding an option feature to use the [`VACUUM INTO`](https://sqlite.org/lang_vacuum.html#vacuuminto) syntax versus the default `.backup` command?

I prefer to use this command to save a bit of additional storage space in my backups

**Comment by efrecon** [comment]
yes. Go on! Maybe offer it as a `SQLITE_BACKUP_COMMAND` (and associated option) that defaults to `.backup` but can be changed to `VACUUM INTO`
